Kattis/Spelling.py

```diff
@@ -0,0 +1,47 @@
+# dictionary for this specific problem
+phonedict = {
+    "a": 2,
+    "b": 22,
+    "c": 222,
+    "d": 3,
+    "e": 33,
+    "f": 333,
+    "g": 4,
+    "h": 44,
+    "i": 444,
+    "j": 5,
+    "k": 55,
+    "l": 555,
+    "m": 6,
+    "n": 66,
+    "o": 666,
+    "p": 7,
+    "q": 77,
+    "r": 777,
+    "s": 7777,
+    "t": 8,
+    "u": 88,
+    "v": 888,
+    "w": 9,
+    "x": 99,
+    "y": 999,
+    "z": 9999,
+    " ": 0,
+}
+
+# taking in multiple x inputs
+n = int(input())
+
+List = []
+
+for i in range(n):
+
+    elem = input()
+
+    List.append(elem)
```

Comment:
Here, you can turn `elem` into a list of `char`, then loop through that list and get the corresponding integer from the dictionary above out.
You can access the value from a dictionary using `[]` operation. For example, `phonedict['a']` would return `2`.